### ISSUE TITLE
Add tray support [Feature]

### ISSUE THREAD

**Issue by anshumanv** [fossasia/susi_desktop#75]
This will keep susi running in the system tray when closed, so it can be accessed quickly.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
